src/NSwag.CodeGeneration.CSharp/Templates/Controller.liquid

```diff
@@ -62,7 +62,7 @@ public partial class {{ Class }}Controller : {% if HasBaseClass %}{{ BaseClass }
     public {% if operation.WrapResponse %}async System.Threading.Tasks.Task<HttpResponseMessage>{% else %}{{ operation.ResultType }}{% endif %} {{ operation.ActualOperationName }}({% for parameter in operation.Parameters %}{% if parameter.IsQuery and IsAspNetCore %}[{{ AspNetNamespace }}.FromQuery{% if parameter.IsValidIdentifier == false %}(Name = "{{ parameter.Name }}"){% endif %}] {% endif %}{% if parameter.IsHeader %}[{% if IsAspNetCore -%}{{ AspNetNamespace }}.{% endif -%}FromHeader{% if parameter.IsValidIdentifier == false %}(Name = "{{ parameter.Name }}"){% endif %}] {% endif %}{% if parameter.IsBody and parameter.IsBinaryBody == false %}[{{ AspNetNamespace }}.FromBody] {% endif %}{% if GenerateModelValidationAttributes and parameter.IsRequired %}[{{ RequiredAttributeType }}] {% endif %}{{ parameter.Type }} {{ parameter.VariableName }}{% if GenerateOptionalParameters and parameter.IsOptional %} = null{% endif %}{% if parameter.IsLast == false or UseCancellationToken %}, {% endif %}{% endfor %}{% if UseCancellationToken %}System.Threading.CancellationToken cancellationToken{% endif %})
     {
 {%         if operation.WrapResponse -%}
-        var result = await _implementation.{{ operation.ActualOperationName }}Async({% for parameter in operation.Parameters %}{{ parameter.VariableName }}{% if parameter.IsLast == false or UseCancellationToken %}, {% endif %}{% endfor %}{% if UseCancellationToken %}cancellationToken{% endif %}).ConfigureAwait(false);
+        var result = await _implementation.{{ operation.ActualOperationName }}Async({% for parameter in operation.Parameters %}{{ parameter.VariableName }}{% if parameter.HasDefault %} ?? {{parameter.Default}}{% endif %}{% if parameter.IsLast == false or UseCancellationToken %}, {% endif %}{% endfor %}{% if UseCancellationToken %}cancellationToken{% endif %}).ConfigureAwait(false);
```

Comment:
Please dont use “new” c# features as some users might not be able to use them - use `... ? ... :` here. Do you agree?

Reply:
Hi Rico,
I've copied it from the same file (line 75). I can change it but that means we also need to cast the nullable types. For example an integer x with a default value of 100.

`int? x = null;`
`x is null ? 100 :  x  //This will fail...`
`x is null ? 100 : (int)x  //This will work...`

What do you prefer?

Reply:
`x != null ? x : 100`?

Reply:
Same: CS1503	Argument 1: cannot convert from 'int?' to 'int'

Reply:
Ok, it's fine for me to keep as is. Can we merge then?